src/bootstrap/builder/tests.rs

```diff
@@ -7,7 +7,16 @@ fn configure(cmd: &str, host: &[&str], target: &[&str]) -> Config {
     // don't save toolstates
     config.save_toolstates = None;
     config.dry_run = true;
+
+    // Ignore most submodules, since we don't need them for a dry run.
+    // But make sure to check out the `doc` and `rust-analyzer` submodules, since some steps need them
+    // just to know which commands to run.
+    let submodule_build = Build::new(Config::parse(&["check".to_owned()]));
+    let submodule_builder = Builder::new(&submodule_build);
```

Comment:
Do we need to initialize Builder here? It looks like update_submodule is defined on Build?

Reply:
Oh good point, simplified it to Build. The difference between the two has always confused me.